client/search-ui/src/results/progress/StreamingProgressSkippedPopover.tsx

```diff
@@ -71,6 +71,8 @@ const SkippedMessage: React.FunctionComponent<React.PropsWithChildren<{ skipped:
                         >
                             <Typography.H4 className="d-flex align-items-center mb-0 w-100">
                                 <Icon
+                                    role="img"
+                                    aria-hidden={true}
```

Comment:
```suggestion
                                    aria-label={skipped.severity === 'info' ? 'Information' : 'Alert'}
```

This one feels important for the user journey